.gitignore

```diff
@@ -7,6 +7,10 @@
 *.tmp
 *.egg-info
 static/
+uploads/*
+!uploads/agencies/
+uploads/agencies/*
+!uploads/agencies/cst-*
```

Comment:
I feel like these can be simplified and we should only need 2: ignore the root folder, and then allow anything matching the `agencies/cst-*` pattern?

Reply:
There are 2 changes that I think will make this work:

1. Remove `*` from the first line: `uploads/` -- this says, ignore the whole directory (with the `*` it means, ignore everything _inside_ the directory, but not the directory itself)
2. Add `.png` to the last line: `!uploads/agencies/cst-*.png` -- this is specifically PNG files in that subdirectory matching the name

Then the other 2 shouldn't be necessary.

Reply:
Ah, sorry for getting stuck on this, I didn't think this would be as complicated since I was also thinking that the simplified version should work:
```
uploads/
!uploads/agencies/cst-*.png
```
but I'm guessing that there's something about having that intermediate `agencies` folder:

Without any additions to the `gitignore`(`uploads` not ignored, all files inside it untracked)
<img src="https://github.com/user-attachments/assets/ed833306-0480-4ecc-8a23-6f1c58b61fd1" width="500" height="288">

With our simplified `gitignore` (all ignored, even CST's logos unfortunately):
<img src="https://github.com/user-attachments/assets/35162b58-ad20-40a0-9871-722549f982d6" width="500" height="272">

With the current PR's `gitignore` (all ignored except CST's logos):
<img src="https://github.com/user-attachments/assets/cdfa9437-c174-4fa8-8959-b99ca8f630fe" width="500" height="286">

Reply:
We decided to go with a separate `.gitignore` file in `uploads` instead of putting everything in the root `.gitignore`